presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/joins/JoinGraph.java

```diff
@@ -14,6 +14,7 @@
 package com.facebook.presto.sql.planner.optimizations.joins;
```

Comment:
Does this commit fix a bug due to the migrated code not being able to operate on GroupReferences? If so, it should be part of the migration commit, as it's inherently part of the work required.